lib/CMakeLists.txt

```diff
@@ -33,7 +33,19 @@ endif()
 
 add_subdirectory(discord)
 
-include(libRocket.cmake)
+option(USE_BUNDLED_LIBROCKET "Use bundled version of librocket instead of system one" ON)
+
+if (USE_BUNDLED_LIBROCKET)
+    IF(NOT IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/libRocket/Build/cmake")
+        message(FATAL_ERROR "libRocket submodule could not be found. Please make sure you have updated your submodules, or set USE_BUNDLED_LIBROCKET to use system version.")
+    endif()
+
+    include(libRocket.cmake)
+else(USE_BUNDLED_LIBROCKET)
+    find_package(libRocket REQUIRED)
+    add_library(libRocket INTERFACE)
+    target_link_libraries(libRocket INTERFACE "${libRocket_LIBRARIES}")
```

Comment:
Not sure how that package is done but does it also have an include directory? Most CMake packages also have that so I maybe that also needs to be added here via `target_include_directories`.